### PR TITLE
Fix subscription namaspace

### PR DESCRIPTION
**Pull request by bamachrn**
Restoring the namespace change to solve degraded state

### CODE REVIEW

File: configs/enable-dvo-for-all-cluster/install-dvo.yaml [hunk 1]
```diff
@@ -20,12 +20,13 @@ metadata:
...
   labels:
     name: deployment-validation-operator
+  namespace: deployment-validation-operator
 spec:
   channel: alpha
   installPlanApproval: Automatic
   name: deployment-validation-operator
   source: community-operators
-  sourceNamespace: deployment-validation-operator
+  sourceNamespace: openshift-marketplace
```
Comment (hugares) [inline configs/enable-dvo-for-all-cluster/install-dvo.yaml:29]: if we use community-operators catalog, do we still need to define our own catalog source(line 1 to 15)?

Comment (hugares) [reply configs/enable-dvo-for-all-cluster/install-dvo.yaml:29]: I tested on my dev cluster without the custom catalog source and the operator installed without any issue.

Comment (bamachrn) [reply configs/enable-dvo-for-all-cluster/install-dvo.yaml:14]: done

**Pull request merged**